### PR TITLE
MRF Version and Validator Version Mismatch 

### PR DESCRIPTION
**Pull request by blakehice4**
https://github.com/CMSgov/price-transparency-guide-validator/issues/121

### CODE REVIEW

File: src/SchemaManager.ts [hunk 1]
```diff
@@ -44,15 +44,25 @@ export class SchemaManager {
     const tags = tagResult.stdout
       .split('\n')
       .map(tag => tag.trim())
-      .filter(tag => tag.length > 0);
-    if (tags.includes(version)) {
-      await util.promisify(exec)(`git -C "${this.repoDirectory}" checkout ${version}`);
-      this._version = version;
+      .filter(tag => tag.length > 0)
```
Comment (blakehice4) [inline src/SchemaManager.ts:47]: image

Comment (blakehice4) [reply src/SchemaManager.ts:47]: image

Comment (blakehice4) [reply src/SchemaManager.ts:47]: image

**Comment by mint-thompson** [comment]
Thank you for opening this PR! This is a very useful change. There are some minor formatting issues that I'll tidy up, and then I think it'll be ready for review by the rest of the team.

**Review by mint-thompson — APPROVED** [review]

**Comment by shaselton-usds** [comment]
@blakehice4 Thank you so much for this! It is much appreciated!

**Pull request merged**